scrapy/crawler.py

```diff
@@ -152,6 +153,8 @@ def start(self, stop_after_crawl=True):
         if self.settings.getbool('DNSCACHE_ENABLED'):
             reactor.installResolver(CachingThreadedResolver(reactor))
 
+        tp = reactor.getThreadPool()
+        tp.adjustPoolsize(maxthreads=self.settings.getint('REACTOR_THREADPOOL_MAXSIZE'))
```

Comment:
adjustPoolSize raises an exception if maxthreads is greater than minthreads. What is the default min value? I can only find "5" in twisted sourc code; does it mean REACTOR_THREADPOOL_MAXSIZE <= 5 are not allowed? It could make sense to decrease min value if user sets REACTOR_THREADPOOL_MAXSIZE to a smaller value.